packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -485,7 +476,7 @@ const StyledTextButton = styled(TextButton)`
  * The mini edit view for a relation that is displayed inside a modal.
  * It's complete with its header, document actions and form layout.
  */
-const RelationEditView = ({ children }: { children: React.ReactNode }) => {
+const RelationEditView = () => {
```

Comment:
Unrelated nitpick, I find RelationEditView misleading, what do you think of RelationEditForm?

Reply:
sure thing 👍